docs/backend-system/building-backends/08-migrating.md

```diff
@@ -177,11 +177,10 @@ custom API, so we use a helper function to transform that particular one.
 To make additions as mentioned above to the environment, you will start to get
 into the weeds of how the backend system wiring works. You'll need to have a
 service reference and a service factory that performs the actual creation of
-your service. Please see [the services
-article](../architecture/03-services.md#defining-a-service) to learn how to
-create a service ref and its default factory. You can place that code directly
-in the index file for now if you want, or near the actual implementation class
-in question.
+your service. Please see [the services article](../architecture/03-services.md)
```

Comment:
Not sure why but Docusaurus didn't seem to like the line break there. Also, I could not find a `defining-a-service` heading so linked to the page it all seems relevant to this topic